nebula_common/include/nebula_common/continental/continental_ars548.hpp

```diff
@@ -292,6 +305,74 @@ constexpr int blockage_test_ongoing = 2;
 constexpr int min_odometry_hz = 10;
 constexpr int max_odometry_hz = 50;
 
+struct FieldInfo
+{
+  FieldInfo(
+    bool min_value_available, bool max_value_available, bool resolution_available, float min_value,
+    float max_value, float resolution)
+  : min_value_available(min_value_available),
+    max_value_available(max_value_available),
+    resolution_available(resolution_available),
+    min_value(min_value),
+    max_value(max_value),
+    resolution(resolution)
+  {
+  }
+  bool min_value_available;
+  bool max_value_available;
+  bool resolution_available;
+  float min_value;
+  float max_value;
+  float resolution;
+};
+
+// Detection field infos
+const FieldInfo azimuth_info{true, true, false, -M_PI, M_PI, 0.f};
+const FieldInfo azimuth_std_info{true, true, false, 0.f, 1.f, 0.f};
+const FieldInfo elevation_info{true, true, false, -M_PI, M_PI, 0.f};
+const FieldInfo elevation_std_info{true, true, false, 0.f, 1.f, 0.f};
+
+const FieldInfo range_info{true, true, false, 0.f, 301.f, 0.f};
```

Comment:
My datasheet is saying `max=1500`, which version are you using?

Reply:
Since this covers technical aspects of the datasheet, I will answer this one via DM

Reply:
Resolved via DM. 301 is the preferred value.